client/src/store/reducers/reportsReducer.js

```diff
@@ -3,11 +3,76 @@ import * as Actions from "store/actions";
 import BaseReducer from "./baseReducer";
 import * as actionTypes from "store/types.js";
 import * as themes from "constants/themes.js";
-
+import * as Validation from "constants/validation.js";
 
 export default class ReportsReducer extends BaseReducer {
   initialState = {
-
+    "ATTENDANCE": {
+      label: "Attendance",
+      key: "ATTENDANCE",
+      jsonEndpoint: "/reportsAttendanceJson",
+      csvEndpoint: "/reportsAttendanceCsv",
+      parametersEndpoint: "/reportsAttendanceParameters",
+      localLink: "/reports/attendance",
+      headers: [],
+      data: [],
+      parameters: [
+        {
+          "key": "startdate",
+          "type": "DATE",
+          "label": "Start date",
+          "description": "This parameter will [blah blah blah]",
```

Comment:
What information is intended to go in the description field? Or, I guess, how does this differ from the tooltip info?

Reply:
The tooltip is a terse instruction for how to use the parameter, while the description is a longer explanation containing anything relevant to the parameter. It doesn't really matter for our two reports since our parameters are pretty self-explanatory, but the point of this format is to be easily extendable.

So, suppose we need to be able to display currencies in multiple units. Then the parameter could be "currency_unit" and the human label "Currency". The tooltip could be: "Currency unit." And the description:

> Controls the units of currency displayed in the report. Possible values are USD, CAD, or EUR. Currency is stored in internally as USD and converted with the newest exchange rate from an online API. The default API is currencyfinex.com; to select a different API, visit the Settings page and click on the Units tab.

Reply:
Cool, makes sense, thanks for clarifying! Since the parameters are all dates at this point, can we go ahead and add a simple description to them before this gets merged in? Sounds like something like "Controls the date at which the report data begins / ends. Only Classes / Sales from before / after this date will be returned. Must be in the format MM/DD/YYYY" would be sufficient for all of the current parameters? Just so we don't accidentally leave a dummy value here.

Reply:
Fixed. If you're okay with it, I'll merge now.